hugegraph-server/hugegraph-test/src/main/java/org/apache/hugegraph/api/BaseApiTest.java

```diff
@@ -84,6 +84,7 @@ public static void init() {
     @AfterClass
     public static void clear() throws Exception {
         client.close();
+        client = null;
```

Comment:
what's for?

Reply:
No special meaning, just set the client member variable to null to indicate that the closed client should not be used anymore.